### ISSUE TITLE
 docker-compose up --build

### ISSUE THREAD

**Issue by italktothewind** [br4chu/docker-compose-maven-plugin#7]
I want to recreate the command  "docker-compose up --build" that rebuilds the images in the compose defined in the following way:

```
  mv-app:
    hostname: mv-app
    build:
      context: .
      dockerfile: myDockerfile
```

Any advice? Thanks!

**Comment by mjagus** [comment]
This is not yet supported unfortunately. This seems like a quite useful feature so I will try to add it as soon as possible.

**Comment by mjagus** [comment]
This is now available in 0.9.0 version of the plugin as a new property of `up` goal called `forceBuild`. You can refer to the README for a bit more info if necessary.

Just update the plugin and set the property in your pom.xml as follows:
```
<configuration>
    <forceBuild>true</forceBuild>
</configuration>
```

**Comment by italktothewind** [comment]
Thanks for adding this feature :)
